cmd/openshift-install/main.go

```diff
@@ -17,11 +18,14 @@ var (
 	ignitionConfigsCommand = kingpin.Command("ignition-configs", "Generate the Ignition Config assets")
 	manifestsCommand       = kingpin.Command("manifests", "Generate the Kubernetes manifests")
 	clusterCommand         = kingpin.Command("cluster", "Create an OpenShift cluster")
+	versionCommand         = kingpin.Command("version", "Print version information and exit")
 
 	destroyCommand = kingpin.Command("destroy-cluster", "Destroy an OpenShift cluster")
 
 	dirFlag  = kingpin.Flag("dir", "assets directory").Default(".").String()
 	logLevel = kingpin.Flag("log-level", "log level (e.g. \"debug\")").Default("warn").Enum("debug", "info", "warn", "error", "fatal", "panic")
+
+	version = "" // set in hack/build.sh
```

Comment:
I like to set this to "was not built correctly" to indicate why a user may see an invalid version.

Reply:
> I like to set this to "was not built correctly"...

Done with 3bc83f9 -> 6d60707.

Reply:
Looks like that went into the wrong commit.